client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html

```diff
@@ -15,7 +15,7 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
                 <div class="section-group">
                     <!-- LANGUAGE -->
                     <div class="section-item">
-                        <div class="form-group no-space">
+                        <div class="form-group p-0 m-0">
```

Comment:
Using `p-0` with `form-group` make sense? I think you can just stick with `m-0`.

Reply:
@feoche

Reply:
Margin or padding should not be necessary here, right? `form-group` has already its own definition of it.